src/parser.rs

```diff
@@ -497,7 +497,7 @@ mod tests {
                 Statement::Return(_) => {
                     // TODO: Check some property of the expression.
                 }
-                _ => panic!("Expected a return statement."),
```

Comment:
Q: What's the motivation for this change? I'm not sure how it's better as adding new statements will mean this needs updating needlessly. I'm sure I'm missing something.

Reply:
Making the pattern match explicit and exhaustive means code takes into account all possible values. Once there are more than two kinds of statement, clippy will stop complaining I think